README.md

```diff
@@ -62,3 +62,18 @@ TRELLO_TOKEN=**********
 TRELLO_BOARD=47ebaa0e00564b548464a9d5fb7c796f
 ```
 
+## testing this app
+
+Before you test this app you need to get your environment ready. To do that install pytest:
+```bash
+python -m pip install pytest.
```

Comment:
Rather than installing this with pip, we should continue to track all of our dependencies with poetry; we should add this to our dependencies (with `poetry add pytest`) and then the tests can be run with `poetry run pytest`